policy/modules/services/apache.te

```diff
@@ -903,6 +912,157 @@ optional_policy(`
+
+manage_dirs_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+append_files_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+create_files_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+read_files_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+setattr_files_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+read_lnk_files_pattern(httpd_ctl_t, httpd_log_t, httpd_log_t)
+logging_log_filetrans(httpd_ctl_t, httpd_log_t, file)
+
+allow httpd_ctl_t httpd_modules_t:dir list_dir_perms;
+mmap_exec_files_pattern(httpd_ctl_t, httpd_modules_t, httpd_modules_t)
+read_files_pattern(httpd_ctl_t, httpd_modules_t, httpd_modules_t)
+read_lnk_files_pattern(httpd_ctl_t, httpd_modules_t, httpd_modules_t)
+
+allow httpd_ctl_t httpd_rotatelogs_t:process signal_perms;
+
+manage_dirs_pattern(httpd_ctl_t, httpd_squirrelmail_t, httpd_squirrelmail_t)
+mmap_manage_files_pattern(httpd_ctl_t, httpd_squirrelmail_t, httpd_squirrelmail_t)
+manage_lnk_files_pattern(httpd_ctl_t, httpd_squirrelmail_t, httpd_squirrelmail_t)
+
+allow httpd_ctl_t httpd_suexec_exec_t:file read_file_perms;
+
+allow httpd_ctl_t httpd_sys_script_t:unix_stream_socket connectto;
+allow httpd_ctl_t httpd_sys_script_t:process signull;
+
+
+manage_dirs_pattern(httpd_ctl_t, httpd_tmp_t, httpd_tmp_t)
+manage_files_pattern(httpd_ctl_t, httpd_tmp_t, httpd_tmp_t)
+allow httpd_ctl_t httpd_tmp_t:file map;
+manage_sock_files_pattern(httpd_ctl_t, httpd_tmp_t, httpd_tmp_t)
+manage_lnk_files_pattern(httpd_ctl_t, httpd_tmp_t, httpd_tmp_t)
+files_tmp_filetrans(httpd_ctl_t, httpd_tmp_t, { file dir lnk_file sock_file })
+userdom_user_tmp_filetrans(httpd_ctl_t, httpd_tmp_t, dir)
+
+manage_dirs_pattern(httpd_ctl_t, httpd_tmpfs_t, httpd_tmpfs_t)
+manage_files_pattern(httpd_ctl_t, httpd_tmpfs_t, httpd_tmpfs_t)
+manage_lnk_files_pattern(httpd_ctl_t, httpd_tmpfs_t, httpd_tmpfs_t)
+manage_fifo_files_pattern(httpd_ctl_t, httpd_tmpfs_t, httpd_tmpfs_t)
+manage_sock_files_pattern(httpd_ctl_t, httpd_tmpfs_t, httpd_tmpfs_t)
+fs_tmpfs_filetrans(httpd_ctl_t, httpd_tmpfs_t, { dir file lnk_file sock_file fifo_file })
+
+manage_dirs_pattern(httpd_ctl_t, httpd_var_lib_t, httpd_var_lib_t)
+mmap_manage_files_pattern(httpd_ctl_t, httpd_var_lib_t, httpd_var_lib_t)
+manage_lnk_files_pattern(httpd_ctl_t, httpd_var_lib_t, httpd_var_lib_t)
+files_var_lib_filetrans(httpd_ctl_t, httpd_var_lib_t, { dir file })
+
+setattr_dirs_pattern(httpd_ctl_t, httpd_runtime_t, httpd_runtime_t)
+manage_dirs_pattern(httpd_ctl_t, httpd_runtime_t, httpd_runtime_t)
+manage_files_pattern(httpd_ctl_t, httpd_runtime_t, httpd_runtime_t)
+manage_sock_files_pattern(httpd_ctl_t, httpd_runtime_t, httpd_runtime_t)
+files_runtime_filetrans(httpd_ctl_t, httpd_runtime_t, { file sock_file dir })
+
+manage_dirs_pattern(httpd_ctl_t, squirrelmail_spool_t, squirrelmail_spool_t)
+manage_files_pattern(httpd_ctl_t, squirrelmail_spool_t, squirrelmail_spool_t)
+manage_lnk_files_pattern(httpd_ctl_t, squirrelmail_spool_t, squirrelmail_spool_t)
```

Comment:
I think this plus especially the port binding makes me reconsider my position...

Overall, I'm beginning to feel this seems too much like `httpd_t` to make it worthwhile. I'm not definitely saying no, but I'm leaning that direction.

Another option would be to move nginx into a new domain.